src/makeChange.js

```diff
@@ -1,3 +1,63 @@
+let keys = {
+  quarters: 0,
+  dimes: 0,
+  nickels: 0,
+  pennies: 0
+}
+let takeQuarter = function(amounTGiven, pricey){
```

Comment:
Why not call them `amountGiven` and `price`?  These variables/parameters are only defined within the scope of this function so you might as well give them readable names.